massa-execution-worker/src/execution.rs

```diff
@@ -1260,37 +1265,85 @@ impl ExecutionState {
             self.mip_store.clone(),
         );
 
-        // Get asynchronous messages to execute
-        let messages = execution_context.take_async_batch(
-            self.config.max_async_gas,
-            self.config.async_msg_cst_gas_cost,
-        );
+        let ts = get_block_slot_timestamp(
+            self.config.thread_count,
+            self.config.t0,
+            self.config.genesis_timestamp,
+            self.final_cursor,
+        )
+        .unwrap();
```

Comment:
Don't we want something else than `unwrap` here?
(returning the error oh at least an `expect`)
(5 other places in the code with same `unwrap`)